pkg/config/config.go

```diff
@@ -113,6 +115,21 @@ func CouchURL() string {
 	return config.CouchDB.URL
 }
 
+var redisClient *redis.Client
+
+// RedisClient returns the global redis client or
```

Comment:
This method should be thread-safe.

Also I am not sure it is the good place to store this global handler.